src/states/node.rs

```diff
@@ -2272,14 +2282,24 @@ impl Bootstrapped for Node {
             None => Ok(()),
             Some(target_name) if target_name == *self.name() => {
                 trace!("{:?} Starting message accumulation for {:?}", self, signed_msg);
-                self.accumulate_message(signed_msg, route)
+                let our_name = *self.name();
```

Comment:
That's `target_name`. (Maybe `target_name` should just be called `our_name` in this match arm.)